_rules/SC2-4-9-links-with-identical-accessible-names-have-same-purpose.md

```diff
@@ -0,0 +1,234 @@
+
+```html
+<a href="/test-assets/link-purpose/about/contact.html">Link text</a>
+<a href="/test-assets/link-purpose/careers/contact.html>Link text</a>
+```
+
+#### Passed example 6
+
+URLs differ due to trailing slashes:
+
+```html
+<a href="/test-assets/link-purpose/link-purpose">Link text</a> 
+<a href="/test-assets/link-purpose/link-purpose/">Link text</a>
+```
+
+#### Passed example 7
+
+Pages contain different amounts of information and/or differently worded information, but fulfils same purpose in relation to the link:
+
+```html
+<a href="/test-assets/link-purpose/page1.html">Link text</a>
+<a href="/test-assets/link-purpose/page2.html">Link text</a>
+```
+
+#### Passed example 8
+
+Has the same content but are using different layouts:
+
+```html
+<a href="/test-assets/link-purpose/page1.html">Link text</a>
+<a href="/test-assets/link-purpose/page3.html">Link text</a>
+```
+
+#### Passed example 9
+
+Links created via scripting with explicit role of link: 
+
+```html
+<span role="link"
+   onclick="location='/test-assets/link-purpose/link-purpose/index.html.html'">
+Link text
+</span>
+
+<span role="link"
+   onclick="location='/test-assets/link-purpose/link-purpose/index.html'">
+Link text
+</span>
+```
+
+### Failed
+
+#### Failed example 1
+
+Same link text used for links going to different resources:
+
+```html
+<a href="http://facebook.com">Follow us</a> 
+<a href="http://twitter.com">Follow us</a>
+```
+
+#### Failed example 2
+
+Same link text used for links going to web pages with same name, but with different information:
+
+```html
+<a href="/test-assets/link-purpose/social-sciences/contact.html">Contact us</a> 
+<a href="/test-assets/link-purpose/humanities/contact.html">Contact us</a>
+```
+
+#### Failed example 3
+
+Case-sensitivity in file name:
+
+```html
+<a href="/test-assets/link-purpose/page1.html">Link text</a> 
+<a href="/test-assets/link-purpose/Page1.html">Link text</a>
+```
+
+#### Failed example 4
+
+Links created via scripting with explicit role of link: 
+
+```html
+<span role="link"
+   onclick="location='/test-assets/link-purpose/link-purpose/page1.html'">
+Link text
+</span>
+
+<span role="link"
+   onclick="location='/test-assets/link-purpose/link-purpose/contact.html'">
+Link text
+</span>
+```
+
+### Inapplicable 
+
+#### Inapplicable example 1
+
+´a´ and ´area´ elements without ´href´ attribute:
+
+```html
+<a>Link text</a>
+<area aria-label="Link text">
+```
+
+#### Inapplicable example 2
+
+No identical link texts:
+
+```html
+<a href="/test-assets/link-purpose">Link text 1</a>
+<a href="/test-assets/link-purpose">Link text 2</a>
+```
+
+#### Inapplicable example 3
+
+Link is not exposed to assistive technologies;
+
+```html
+<a href="/test-assets/link-purpose" style="display:none">Link text 1</a>
+<a href="/test-assets/link-purpose">Link text 2</a>
+```
+
+#### Inapplicable example 4
+
+Links created via scripting, but doesn't have semantic role of link:
```

Comment:
replace "but doesn't have semantic role of link" with "but without the semantic role of link"